### ISSUE TITLE
Customizing via `config.cson` doesn't work

### ISSUE THREAD

**Issue by shaungrady** [atom-community/autocomplete-paths#120]
The small example in the README doesn't seem to work. The closest "shape" I could get Atom to accept is this:

``` cson
  "autocomplete-paths":
    extensions: [
      "js"
      "ts"
      "html"
    ]
    includeCurrentDirectory: true
    prefixes: [
      "import\\s+.*?from\\s+['\"]"
    ]
    relative: true
    replaceOnInsert: [
      [
        ".jsx?$"
        ""
      ]
      [
        ".ts$"
        ""
      ]
      [
        ".coffee$"
        ""
      ]
    ]
    scopes: []
```

Notice that the `scopes` array is empty. Couldn't figure out what could make it happy.

**Comment by DamnedScholar** [comment]
Take a look at the examples again. You have `scopes`, which is a list of dictionaries. Each dictionary has a `scopes` key that tells it what to check. The two different `scopes` keys are why you're confused. Here's the correct format:
```
  "autocomplete-paths":
    scopes: [
      {
        scopes: [
          "source.python"
        ]
        prefixes: [
          "import"
          "from"
        ]
        extensions: [
          "py"
          "pyc"
        ]
        relative: true
        includeCurrentDirectory: true
        replaceOnInsert: [
          [
            ".py?$"
            ""
          ]
          [
            ".pyc$"
            ""
          ]
        ]
      }
    ]
```

**Comment by shaungrady** [comment]
Thank you, @DamnedScholar, that was it. 🙇  It's been a long day :)

**Comment by DamnedScholar** [comment]
The examples are not the most straightforward. I noticed that they seemed to fit together, but they used the same key name, so I took a bet that it was just repetition and went with that and it worked.

**Comment by cbrwizard** [comment]
I also have some problems here. I'm trying to set the `relative` to false for js files. I am editing `config.cson` like this:
```
  "autocomplete-paths":
    scopes: [
      {
        scopes: ['source.js', 'source.js.jsx', 'source.coffee', 'source.coffee.jsx', 'source.ts', 'source.tsx'],
        prefixes: [
          'import\\s+.*?from\\s+[\'"]',
          'require\\([\'"]',
          'define\\(\\[?[\'"]'
        ],
        extensions: ['js', 'jsx', 'ts', 'tsx', 'coffee'],
        relative: false,
        replaceOnInsert: [
          ['.jsx?$', ''],
          ['.ts$', ''],
          ['.coffee$', '']
        ]
      },
    ]
```
But I'm still having relative paths inserted when I autocomplete import statements in js files. Tried restarting Atom. Maybe I configured it wrong?

**Comment by DamnedScholar** [comment]
@cbrwizard What does your `config.cson` look like after you restart Atom?

**Comment by saschagehlich** [comment]
@cbrwizard Just released 2.8.0 which makes user scopes override the default scopes
